pkgs/build-support/cc-wrapper/add-clang-cc-cflags-before.sh

```diff
@@ -1,15 +1,24 @@
 needsTarget=true
+targetValue=""
 
 for p in "${params[@]}"; do
     case "$p" in
-    -target | --target=*)
+    -target)
         needsTarget=false
-
-        echo "Warning: supplying the --target argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead." >&2
+        targetValue=$2
+        shift 2
+        ;;
+    --target=*)
+        needsTarget=false
+        targetValue="${p#*=}"
         ;;
     esac
 done
 
+if ! $needsTarget && [[ "$targetValue" != "@defaultTarget@" ]]; then
+    echo "Warning: supplying the --target argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead." >&2
```

Comment:
It might be nice to print `$targetValue` now that we know what it is, to aid debugging.